ompi/mpi/man/man3/MPI_Cart_rank.md

```diff
@@ -0,0 +1,72 @@
+# Name
+
+`MPI_Cart_rank` - Determines process rank in communicator given
+Cartesian location.
+
+# Syntax
+
+## C Syntax
+
+```c
+#include <mpi.h>
+
+int MPI_Cart_rank(MPI_Comm comm, int coords[], int *rank)
+```
+
+## Fortran Syntax
+
+```fortran
+USE MPI
+! or the older form: INCLUDE 'mpif.h'
+
+MPI_CART_RANK(COMM, COORDS, RANK, IERROR)
+    INTEGER	COMM, COORDS(*), RANK, IERROR
+```
+
+## Fortran 2008 Syntax
+
+```fortran
+USE mpi_f08
+
+MPI_Cart_rank(comm, coords, rank, ierror)
+    TYPE(MPI_Comm), INTENT(IN) :: comm
+    INTEGER, INTENT(IN) :: coords(*)
+    INTEGER, INTENT(OUT) :: rank
+    INTEGER, OPTIONAL, INTENT(OUT) :: ierror
+```
+
+# Input Parameters
+
+* `comm` : Communicator with Cartesian structure (handle).
+* `coords` : Integer array (of size ndims, which was defined by MPI_Cart_create
+call) specifying the Cartesian coordinates of a process.
+
+# Output Parameter
+
+* `rank` : Rank of specified process (integer).
+* `IERROR` : Fortran only: Error status (integer).
+
+# Description
+
+For a process group with Cartesian structure, the function `MPI_Cart_rank`
+translates the logical process coordinates to process `rank`s as they are
+used by the point-to-point routines. For dimension i with periods(i) =
+true, if the coordinate, `coords`(i), is out of range, that is, `coords`(i)
+< 0 or `coords`(i) >= dims(i), it is shifted back to the interval 0 =<
+`coords`(i) < dims(i) automatically. Out-of-range coordinates are
```

Comment:
Note that this paragraph actually chose to express itself in Fortran notation -- `periods(i)` and `coords(i)` and `dims(i)` (vs. `periods[i]` and `coords[i]` and `dims[i]`).

Can you either include the `(i)` in the quoting, or convert it to C-style and include the `[i]` in the quoting?